src/helper.js

```diff
@@ -697,6 +697,15 @@ class helper {
 	static getStartListIndex = (pageNumber, pageSize) => {
 		return pageNumber === 1 ? 0 : (pageNumber - 1) * pageSize;
 	}
+
+	static getInflationByHeight = (height) => {
+		const blocksHeight = Object.keys(inflation);
+
+		for (let i = 0; i < blocksHeight.length; i++) {
```

Comment:
1. `length - 1` or else last iteration will try to index length because of `i + 1`.
2. please use `++i` instead of `i++` whenever possible; former can generally be better optimized (although compilers/interpreters should be smart enough nowadays)
3. you should refer last value if height is past the last entry

Reply:
In this method, I will remove and use the receipt endpoint.
Btw that is good feedback for me to improve the next method.